plugins/modules/source_control/gitlab/gitlab_project.py

```diff
@@ -396,7 +458,14 @@ def main():
                                                 "snippets_enabled": snippets_enabled,
                                                 "visibility": visibility,
                                                 "import_url": import_url,
-                                                "lfs_enabled": lfs_enabled}):
+                                                "lfs_enabled": lfs_enabled,
+                                                "allow_merge_on_skipped_pipeline": allow_merge_on_skipped_pipeline,
+                                                "only_allow_merge_if_all_discussions_are_resolved": only_allow_merge_if_all_discussions_are_resolved,
+                                                "only_allow_merge_if_pipeline_succeeds": only_allow_merge_if_pipeline_succeeds,
+                                                "packages_enabled": packages_enabled,
+                                                "remove_source_branch_after_merge": remove_source_branch_after_merge,
+                                                "squash_option": squash_option
```

Comment:
```suggestion
                                                "squash_option": squash_option,
```
Makes the diff of the next change smaller :)